src/components/gauges/HorizontalGauge.jsx

```diff
@@ -70,30 +71,42 @@ const HorizontalGauge = React.createClass({
   },
 
   render() {
-    let { stacks, all, style, ...otherProps } = this.props;
+    let { stacks, all, style, renderTooltip, className, ...otherProps } = this.props;
     let { hasTooltip } = this.state;
+
     return (
       <FollowCursor
         hasOverlay={hasTooltip}
-        renderOverlay={this._renderTooltip}
+        renderOverlay={renderTooltip || this._renderTooltip}
```

Comment:
This, along with wrapping all stacks in a div, should allow us to use HorizontalGauge in other places without having to create another component.